### DISCUSSION TITLE
Z2M won't find anything after changing zigbee channel

### DISCUSSION THREAD

**Discussion by Wouterz90** [Koenkk/zigbee2mqtt#17814] [Devices]
I was a very satisfied with Z2M but noticed that a few devices would go offline without reason and I thought it might interfere with some WiFi signals which were also at channel 11. So I changed it to 25, thought I would have to re-pair all devices, but after I deleted them, they just won't show up anymore. I tried reflashing my SONOFF Zigbee 3.0 USB Dongle Plus ZBDongle-P, but that didn't solve the issue either.

The log doesn't give any information on this at all.

configuraiton.yaml
`homeassistant: true
mqtt:
  server: mqtt://core-mosquitto:1883
  user: mqtt_user
  password: een
serial:
  port: >-
    /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_fe5ae706009dec11b5f56a4d73138bba-if00-port0
frontend:
  port: 8099
advanced:
  homeassistant_legacy_entity_attributes: false
  legacy_api: false
  legacy_availability_payload: false
  log_syslog:
    app_name: Zigbee2MQTT
    eol: /n
    host: localhost
    localhost: localhost
    path: /dev/log
    pid: process.pid
    port: 514
    protocol: udp4
    type: '5424'
  channel: 25
  log_level: debug
device_options:
  legacy: false
permit_join: true
devices: {}
availability: true`

Am I overlooking something simple? Anyone familiar with this?

**Comment by lux73** [comment]
Zigbee Channel Map != Wifi Channel Map - so Zigbee Channel 11 equals to Wifi Channel 1 (weird i know)

https://www.metageek.com/training/resources/zigbee-wifi-coexistence/

please don't use Zigbee Channel higher than 25 - some Devices could simply not connect to Chan over 25

**Reply by Wouterz90** [reply]
Thanks, I should have mentioned that I've alreadty tried setting it back to 11, doesn't work either. The devices act like they get paired correctly (stop blinking/showing pair message) but they never show up. Could it be that someone else nearby always allows devices and somehow gets priority over me?

**Reply by lux73** [reply]
possible but unlikely
